### PR TITLE
Fix a few links

### PR DESCRIPTION
**Pull request by eviau-artefactual**
As part of https://github.com/archivematica/Issues/issues/708 and https://github.com/archivematica/Issues/issues/1638

### CODE REVIEW

File: admin-manual/installation-setup/customization/antivirus-admin.rst [hunk 1]
```diff
@@ -159,5 +159,5 @@ environment itself.
 .. _ClamAV manual pages: https://manpages.debian.org/jessie/clamav-daemon/clamd.conf.5.en.html
 .. _Compose .yml file: https://github.com/artefactual/archivematica/blob/qa/1.x/hack/docker-compose.yml
 .. _configuration file: https://github.com/artefactual-labs/ansible-clamav/blob/master/defaults/main.yml
-.. _Ansible environment variables: https://github.com/artefactual-labs/ansible-archivematica-src/tree/d4474c3dbaef2b561c87e0650c6ee386be6910a7#environment-variables
+.. _Ansible environment variables: https://github.com/artefactual-labs/ansible-archivematica-src/blob/stable/1.15.x/README.md
```
Comment (replaceafill) [inline admin-manual/installation-setup/customization/antivirus-admin.rst]:
```suggestion
.. _Ansible environment variables: https://github.com/artefactual-labs/ansible-archivematica-src/blob/stable/1.15.x/README.md#environment-variables
```

**Review by replaceafill — APPROVED** [review]

File: admin-manual/installation-setup/customization/antivirus-admin.rst (hunk 1 above)
Comment (eviau-artefactual) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: I tried this, and it makes `make linkcheck` fail :/ 

Either we could find the regex to enable headings, or we could skip the headings. I am not sure what the regex would look like ? the pattern is probably `.md`+ `#` + `combination of letters, numbers and dashes` but I am probably wrong :sweat_smile:

Comment (replaceafill) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: Hhmm depending on how many links use fragments across the docs maybe it's worth figuring it out.

Comment (eviau-artefactual) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: Oh I think I got it :sparkles: 

`r"((.md)[#][\w-]*)"`

Added in https://github.com/artefactual/archivematica-docs/pull/440/commits/68eec5bc84ed1412d8d33f5224f272f3cff86c25

Comment (eviau-artefactual) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: Hmm - it's not quite it yet. :construction:

Comment (eviau-artefactual) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: ok now I think I got it, after some more testing - https://github.com/artefactual/archivematica-docs/pull/440/commits/4edd720320fa60be1d05ee0d1f2ac31379451fdf#diff-6e03d3d89630aba3403afea3514a3ac5948e70d1ced091012e56446f614da340R377

Comment (replaceafill) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: :exploding_head: 

Great. Please don't forget to squash and merge.

**Pull request merged**